actors/common/src/main/java/org/sunbird/learner/actors/role/dao/impl/RoleDaoImpl.java

```diff
@@ -9,15 +9,15 @@
 import org.sunbird.common.models.util.JsonKey;
 import org.sunbird.helper.ServiceFactory;
 import org.sunbird.learner.actors.role.dao.RoleDao;
+import org.sunbird.learner.util.Util;
 import org.sunbird.models.role.Role;
 
 public class RoleDaoImpl implements RoleDao {
 
   private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
   private ObjectMapper mapper = new ObjectMapper();
   private static RoleDao roleDao;
-  private static final String KEYSPACE_NAME = "sunbird";
-  private static final String TABLE_NAME = "role";
+  private final Util.DbInfo roleDbInfo  = Util.dbInfoMap.get(JsonKey.ROLE);
```

Comment:
Suggestion is to move away from maintaining a static list of db info map in memory. The table name can be defined locally as a constant in this class itself. The keyspace name can be a global constant shared by all Dao classes.